tests/benchmark.txt

```diff
@@ -1,1254 +1,1269 @@
 GTFS feed saved in /home/mehdi/Projects/hiposfer/osmtogtfs/tests/out
-         7526061 function calls (7523741 primitive calls) in 23.256 seconds
+         3751892 function calls (3749564 primitive calls) in 16.397 seconds
```

Comment:
@mehdisadeghi nice improvement :)

really nice to be able to identify it inmediately

Reply:
@carocad yeah, actually this is a simple way that I saw you using in `kamal` so I followed the suit! :)
BTW, I realized that memory profiling in the script is as important as the speed tests. I was unable to process Hessen data due to insufficient memory. The main problem is the huge number of nodes which I have to store them for further processing. I digged a bit and realized Osmium has special support for some stores and high performance processing which I have yet to discover.

Reply:
yeah, this is why I switched from custom osm preprocessing to overpass-api query. By leaving the filtering outside of the scope I can focus on data processing instead of joggling with memory issues.

> BTW, I realized that memory profiling in the script is as important as the speed tests. I was unable to process Hessen data due to insufficient memory